src/dlgTriggerEditor.cpp

```diff
@@ -8833,6 +8868,8 @@ QString dlgTriggerEditor::generateButtonStyleSheet(const QColor& color, const bo
         QColor disabledColor = QColor::fromHsl(color.hslHue(), color.hslSaturation()/4, color.lightness());
         return QStringLiteral("QPushButton {color: %1; background-color: %2; }")
                 .arg(QLatin1String("darkGray"), disabledColor.name());
+    } else {
+        return QStringLiteral("QPushButton {color: white; background-color: grey; }");
```

Comment:
What about the dark/light desktop theme differences - will this be right for BOTH cases? :confused:

Reply:
Yep, see attached screenshots.